src/components/index/IndexComponent.tsx

```diff
@@ -71,7 +71,15 @@ const Index = ({
       disputed: [],
       answered: [],
     };
+    const nowTimestamp = parseInt((new Date().getTime() / 1000).toFixed(0));
+    // subtract the seconds for 1 month
+    const lastMonthTimestamp = nowTimestamp - 60 * 60 * 24 * 31;
     const nextFR = descendingRequests.reduce((result, request) => {
```

Comment:
Do we want to move this out of the component itself? It won't re-render obviously, but the lastMonthTimestamp shouldn't really change much when a user's on the page.